src/IB/IBFESurfaceMethod.cpp

```diff
@@ -566,23 +743,18 @@ IBFESurfaceMethod::interpolateVelocity(const int u_data_idx,
         dphi_dxi[0] = &fe->get_dphidxi();
         if (NDIM > 2) dphi_dxi[1] = &fe->get_dphideta();
 
+        FEType fe2_type = DU_jump_fe_type;
+        UniquePtr<FEBase> fe2 = FEBase::build(dim, fe2_type);
+        const std::vector<double>& JxW2 = fe2->get_JxW();
```

Comment:
You don't need this quadrature weight if you are using the same integration rule for both FE families (which I think you are).